README.md

```diff
@@ -16,45 +16,31 @@
 <img alt-text="Iter8 experiment" src="https://iter8-tools.github.io/docs/0.9/images/iter8-intro-dark.png" width="70%" />
 </p>
 
-## :rocket: Getting started
-
-### Install Iter8 CLI
-```shell
-brew tap iter8-tools/iter8
-brew install iter8@0.9
-```
-[See here](https://iter8.tools/latest/getting-started/install) for more ways to install.
-
-### Benchmark an HTTP service
-Launch Iter8 experiment and view report.
-```shell
-iter8 launch -c load-test-http --set url=https://httpbin.org/get
-iter8 report
-```
-
-### Benchmark a gRPC service
-Start a sample gRPC service in a separate terminal.
-
-```shell
-docker run -p 50051:50051 docker.io/grpc/java-example-hostname:latest
-```
-
-Launch Iter8 experiment and view report.
-```shell
-iter8 launch -c load-test-grpc \
---set host="127.0.0.1:50051" \
---set call="helloworld.Greeter.SayHello" \
---set protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/helloworld/helloworld/helloworld.proto"
-iter8 report
-```
+## :rocket: Features at a glance
+
+**Load testing with SLOs**
+
+  > Iter8 experiments can generate requests for HTTP and gRPC services, collect built-in latency and error-related metrics, and validate SLOs.
+
+**A/B(/n) testing**
+
+  > Grow your business with every release. Iter8 experiments can compare multiple versions based on business value and identify a winner.
+
+**Simple to use**
+
+  > Get started with Iter8 in seconds using pre-packaged experiment charts. Run Iter8 experiments locally, inside Kubernetes, or inside your CI/CD/GitOps pipelines.
+
+**App frameworks**
+
+  > Use with any app, serverless, or ML framework. Iter8 works with Kubernetes deployments, statefulsets, Knative services, KServe/Seldon ML deployments, or other custom Kubernetes resource types.
```

Comment:
```suggestion
   > Use with any app, serverless, or ML framework. Iter8 works with Kubernetes deployments, statefulSets, Knative services, KServe/Seldon ML deployments, or other custom Kubernetes resource types.
```

Reply:
Is the extra space in front of `>` intentional?

Reply:
It seems like I can commit your changes to my PR directly through the UI without having to do it in my local repo... hence the question.